src/main/java/com/blamejared/brackets/MaterialBracketHandler.java

```diff
@@ -0,0 +1,86 @@
+package com.blamejared.brackets;
+
+import minetweaker.IBracketHandler;
+import minetweaker.MineTweakerAPI;
+import minetweaker.annotations.BracketHandler;
+import minetweaker.api.item.IngredientAny;
+import com.blamejared.brackets.util.IMaterial;
+import com.blamejared.brackets.util.MCMaterial;
+import slimeknights.tconstruct.library.TinkerRegistry;
+import stanhebben.zenscript.compiler.IEnvironmentGlobal;
+import stanhebben.zenscript.expression.ExpressionCallStatic;
+import stanhebben.zenscript.expression.ExpressionString;
+import stanhebben.zenscript.expression.partial.IPartialExpression;
+import stanhebben.zenscript.parser.Token;
+import stanhebben.zenscript.symbols.IZenSymbol;
+import stanhebben.zenscript.type.natives.IJavaMethod;
+import stanhebben.zenscript.util.ZenPosition;
+
+import java.util.List;
+
+/**
+ * Created by Jared on 6/16/2016.
+ */
+@BracketHandler(priority = 100)
+public class MaterialBracketHandler implements IBracketHandler {
+
+    public static IMaterial getMaterial(String name) {
+        return new MCMaterial(TinkerRegistry.getMaterial(name));
+    }
+
+    private final IZenSymbol symbolAny;
+    private final IJavaMethod method;
+
+    public MaterialBracketHandler() {
+
+        symbolAny = MineTweakerAPI.getJavaStaticFieldSymbol(IngredientAny.class, "INSTANCE");
+        method = MineTweakerAPI.getJavaMethod(MaterialBracketHandler.class, "getMaterial", String.class);
+    }
+
+    @Override
+    public IZenSymbol resolve(IEnvironmentGlobal environment, List<Token> tokens) {
+        // any symbol
+        if (tokens.size() == 1 && tokens.get(0).getValue().equals("*")) {
+            return symbolAny;
```

Comment:
The symbol any is really only for Items, it is not recommended to have it in other bracket handlers (and it *should* actually be it's own bracket handler, but it has been phased out in 1.12 anyway